tests/php/BlockTypes/ProductQuery.php

```diff
@@ -513,5 +530,80 @@ public function test_merging_multiple_filter_queries() {
 		set_query_var( 'min_price', '' );
 		set_query_var( 'filter_stock_status', '' );
 	}
-}
 
+	/**
+	 * Test merging multiple filter queries.
+	 */
+	public function test_updating_rest_query_without_attributes() {
+		$args    = array();
+		$request = $this->build_request();
+
+		$updated_query = $this->block_instance->update_rest_query( $args, $request );
+
+		$this->assertContainsEquals(
+			array(
+				'key'     => '_stock_status',
+				'value'   => array(),
+				'compare' => 'IN',
+			),
+			$updated_query['meta_query'],
+		);
+
+		$this->assertEquals(
+			array(
+				array(
```

Comment:
Same here and the other tests function, I would change all the `array()` by the shorthand syntax.

Reply:
Seems like there are other lint settings in the `tests` directory. Autoformatter changes `[]` to `array()` straight away. Also, considering that all the other tests are written using the `array()` syntax, I would keep it for consistency.

But other than that, I'll be using it for sure! 🙌

Reply:
Ah, that's right, I remember that happened to me too when writing some tests! 👍